apps/backend/api/graphql/mutations/post.js

```diff
@@ -30,9 +30,17 @@ export function deletePost (userId, postId) {
       if (post.get('user_id') !== userId) {
         throw new GraphQLError("You don't have permission to modify this post")
       }
+      return post.isEvent() ? cancelEvent(post) : true
+    }).then(() => {
       return Post.deactivate(postId)
-    })
-    .then(() => ({ success: true }))
+    }).then(() => ({ success: true }))
+}
+
+export async function cancelEvent (post) {
+  await post.load('eventInvitations')
+  const eventInvitations = await post.relations.eventInvitations.fetch()
```

Comment:
this is weird, you shouldn't have to do a .load and a .fetch, isn't that loading eventInvitations twice? why not just do await post.eventInvitations().fetch()?

Reply:
also is there a reason you load these ids here and pass them into sendEventCancelRsvps? seems like sendEventCancelRsvps could just load them from the post itself and that would make more sense to me